packages/dds/sequence/package.json

```diff
@@ -105,38 +105,13 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "1.0.0",
+    "version": "1.1.0",
     "broken": {
-      "InterfaceDeclaration_ISharedString": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SharedNumberSequence": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SharedObjectSequence": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SharedSegmentSequence": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SharedSequence": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SharedString": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_SparseMatrix": {
-        "forwardCompat": false
-      },
       "ClassDeclaration_IntervalCollection": {
         "forwardCompat": false
       },
       "ClassDeclaration_SequenceInterval": {
-        "backCompat": false,
         "forwardCompat": false
```

Comment:
@ransomr just FYI. I know you brought some changes via https://github.com/microsoft/FluidFramework/pull/10576/files. The type validation is against 1.0.0 now (released early last week).